src/components/MatchPredictionsBar.vue

```diff
@@ -12,6 +12,7 @@
     </div>
 
     <div
+      v-if="match.groupId"
       ref="draw"
       :class="['min-w-[2.5em] h-full border-gray-300 border-r border-l']"
```

Comment:
Just a side note, but you usually don't need a `ref`, and can actually add conditional classes in Vue.js like that:

```suggestion
      class="min-w-[2.5em] h-full border-gray-300 border-r border-l"
      :class="result == 'home' ? 'bg-prediction-correct' : 'bg-gray-100'"
```
You could also pass an object to the `:class` attribute, like this:
```js
:class="{ 'conditional-class-name another-class': addClassesIfThisReturnsTrue, ... }"
```

You could also add a dynamic style directly in the template:
```suggestion
      class="min-w-[2.5em] h-full border-gray-300 border-r border-l"
      :class="result == 'home' ? 'bg-prediction-correct' : 'bg-gray-100'"
      :style="`width: ${precentageHomeWin}`"
```

Not tested, but normally with these, you don't need to rely on `refs` or on the `mounted` hook.

Reply:
@trouni The reason I used the ref is because when I originally coded this I was setting the width of the inner bars based on the outer contain bar's width (which I used a ref to calculate). I didn't actually need to do this in the end so the ref is not needed as you pointed out - will refactor :)